src/docs/sdk/envelopes.mdx

```diff
@@ -465,6 +465,26 @@ The item contains a user feedback / user report JSON payload:
 
 *None*
 
+### Client Report
+
+Item type `"client_report"` contains a client report payload encoded in JSON.
+
+See the <Link to="/sdk/client-reports/">client reports</Link> documentation for the payload
+details.
+
+**Constraints:**
+
+- This Item may occur at most once per Envelope.
```

Comment:
Is this correct, or could multiple client reports be sent in a single envelope?

Reply:
I'm not sure. @jan-auer or @untitaker, I'm pretty sure one of you can answer that.

Reply:
You can technically send multiple client reports. If it makes things easier for e.g. multi-language SDKs, sure, but generally please try to avoid sending more client reports than necessary.